files/docker/daemon.json

```diff
@@ -0,0 +1,3 @@
+{
+    "userns-remap": "admin"
```

Comment:
Is this the name of our NS?

Reply:
No, `admin` is not the name of a namespace, but rather the username for the user namespace remapping in Docker